# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/cppu-dmake.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./cppu/util/makefile.mk.orig	2007-07-18 14:34:25.000000000 +0200
+++ ./cppu/util/makefile.mk	2007-12-06 21:27:19.000000000 +0100
@@ -103,7 +103,7 @@
 
 
 ALLTAR:   $(SHL2TARGETN)
-	$(MAKE) $(MAKECMDGOALS) -f extra.mk
+	dmake $(MAKECMDGOALS) -f extra.mk
 
 
 .INCLUDE :	target.mk
